privacy-on-beam/pbeam/count.go

```diff
@@ -68,37 +68,52 @@ type CountParams struct {
 // This aggregation is not hardened for such applications yet.
 //
 // Count transforms a PrivatePCollection<V> into a PCollection<V, int64>.
-func Count(s beam.Scope, pcol PrivatePCollection, params CountParams) beam.PCollection {
-	s = s.Scope("pbeam.Count")
+func Count(s beam.Scope, pcol PrivatePCollection, params CountParams, partitions ...beam.PCollection) beam.PCollection {
```

Comment:
Would be nice to update the function comment here to explain that you can only specify at most one partition